.github/workflows/ci.yml

```diff
@@ -89,43 +89,29 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-20.04, ubuntu-22.04]
-        containerd: [v1.6.21, v1.7.1]
+        containerd: [1.6.21, 1.7.1]
```

Comment:
The release archive name does not have the preceding `v` so I removed it from the matrix and added it in the appropriate places below

Reply:
Hmm.. 
I think archive name can be simplified to something like `--pattern '*-linux-amd64.tar.gz'` (as release version is provided separately via `gh release download ${{ matrix.containerd }} ... `).

Reply:
I tried several different variations and no matter what it said there was unable to download since there is multiple matches. Setting it to the full archive name was the only one that worked
<img width="829" alt="image" src="https://github.com/containerd/rust-extensions/assets/10913471/5052b0d4-7f7c-40cc-b718-361b43a4f864">

Reply:
actually, I found one that works. let me update `gh release download v1.7.3 --repo containerd/containerd --pattern 'containerd-*-linux-amd64.tar.gz' --output containerd.tar.gz`

Reply:
ok updated in https://github.com/containerd/rust-extensions/pull/203/commits/395f7374df3a3ad1d226aada932e38348f56fae8